style.css

```diff
@@ -0,0 +1,205 @@
+html {
+  font-size: 16px;
+  font-family: 'Fira Sans', sans-serif;
+}
+
+body {
+  background-color: #eaebf0;
+  background-image: url(img/background.png);
+  background-size: 20px 10px;
+}
+
+.wrapper {
+  max-width: 1184px;
+  margin: 0px auto;
+}
+
+.main-header {
+  background: #fff;
+}
+
+.main-header-wrapper {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+     -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+     -ms-flex-align: center;
+        align-items: center;
+}
+
+.compass-logo {
+  margin: 1.25rem 0px;
+  width: 179px;
+  height: 24px;
+}
+
+.menu-item {
+  color: #3e4247;
+  text-decoration: none;
+}
+
+.menu-item:not(:last-child) {
+  margin-right: 2rem;
+}
+
+.development-promo {
+  background-color: #161616
+}
+
+.development-logo {
+  display: block;
+  margin: 0px auto;
+  padding: 5rem 0px;
+  width: 419px;
+  height: 131px;
+}
+
+.catalog {
+  padding: 4rem 0px 6rem;
+}
+
+.intro {
+  text-align: center;
+  color: #3e4247;
+  margin-bottom: 4rem;
+}
+
+.intro-title {
+  margin: 0px;
+  font-size: 1.5rem;
+  font-weight: 700;
+  line-height: 2.5rem;
+}
+
+.intro-text {
+  margin: .5rem 0px 0px;
+  line-height: 1.5rem;
+}
+
+.intro-button {
+  background-color: #000;
+  border: none;
+  border-radius: 2px;
+  color: #fff;
+  padding: .8125rem 2rem;
+  margin-top: 1rem;
+  font-family: 'Fira Sans', sans-serif;
+  font-size: 1rem;
+}
+
+.offering {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+     -ms-flex-pack: justify;
+  justify-content: space-between;
+  margin: 3rem auto 0px;
+  max-width: 1184px;
+  background: #fff;
+  text-decoration: none;
+  transition: .2s ease-in-out;
+}
+
+.offering:hover {
+  box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, .3);
+}
+
+.offering-preview {
+  width: 484px;
+  height: 350px;
+  ms-flex-negative: 0;
+  flex-shrink: 0;
+}
+
+.offering-info {
+  display: block;
+  padding-left: 1rem;
+  padding-right: 6.25rem;
+  border-bottom: solid 2px #646971;
+}
+
+.offering-location {
+  display: block;
+  font-family: monaco, Consolas, 'Lucida Console', monospace;
+  color: #646971;
+  margin-top: 1.5rem;
+}
+
+.offering-name {
+  display: block;
+  margin: 1.5rem 0px 0px;
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.5rem;
+  color: #000;
+}
+
+.offering-desc {
+  display: block;
+  margin-top: .5rem;
+  line-height: 1.5rem;
+  color: #3e4247;
+}
+
+.main-footer {
+  background-color: #111;
+  color: #a9afb6;
+  padding: 2rem 0px;
+  font-size: .875rem;
+  line-height: 1.5rem;
+  font-weight: 300;
+}
+
+.footer-header {

```

Comment:
There are only two hard things in Computer Science: cache invalidation and naming things.

-- Phil Karlton